### PR TITLE
[Network Manager] Fix multi issues on network-manager

### PR DESCRIPTION
**Pull request by 00Kai0**
closed https://github.com/Azure/azure-cli/issues/20099

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Network Manager

File: src/network-manager/azext_network_manager/_help.py [hunk 1]
```diff
@@ -289,7 +298,7 @@
                az network manager group create --name "TestNetworkGroup" --network-manager-name "testNetworkManager" \
 --description "A sample group" --conditional-membership "" --display-name "My Network Group" --group-members \
 resource-id="/subscriptions/subscriptionC/resourceGroup/rg1/providers/Microsoft.Network/virtualnetworks/vnet1" \
---member-type "VirtualNetwork" --resource-group "rg1"
+--member-type "Microsoft.Network/virtualNetworks" --resource-group "rg1"
```
Comment (yanfa317) [inline src/network-manager/azext_network_manager/_help.py:301]: "/Microsoft.Network/virtualNetworks"

File: src/network-manager/azext_network_manager/_params.py [hunk 2]
```diff
@@ -228,7 +234,7 @@ def load_arguments(self, _):
                    'changes.')
         c.argument('display_name', type=str, help='A friendly name for the network group.')
         c.argument('description', type=str, help='A description of the network group.')
-        c.argument('member_type', arg_type=get_enum_type(['VirtualNetwork', 'Subnet']), help='Group member type.')
+        c.argument('member_type', arg_type=get_enum_type(['Microsoft.Network/virtualNetworks', 'Microsoft.Network/virtualNetworks/subnets']), help='Group member type.')
```
Comment (yanfa317) [inline src/network-manager/azext_network_manager/_params.py:237]: '/Microsoft.Network/virtualNetworks', '/Microsoft.Network/virtualNetworks/subnets'

File: src/network-manager/azext_network_manager/tests/latest/recordings/test_network_manager_connect_config_crud.yaml [hunk 3]
```diff
@@ -134,15 +135,15 @@ interactions:
       - application/json
       ParameterSetName:
       - --configuration-name --network-manager-name -g --applies-to-groups --connectivity-topology
-        --delete-existing-peering --hubs --description --is-global
+        --delete-peering --hubs --description --is-global
```
Comment (yanfa317) [inline src/network-manager/azext_network_manager/tests/latest/recordings/test_network_manager_connect_config_crud.yaml:138]: Why change this? It should be delete-existing-peering

File: src/network-manager/azext_network_manager/tests/latest/test_network_scenario.py [hunk 4]
```diff
@@ -374,6 +374,7 @@ def test_network_manager_list_queries(self, virtual_network, resource_group):
         self.cmd('network manager list-effect-vnet --network-manager-name {manager_name} --resource-group {rg}')
         self.cmd('network manager list-active-config --network-manager-name {manager_name} --resource-group {rg}')
```
Comment (yanfa317) [inline src/network-manager/azext_network_manager/tests/latest/test_network_scenario.py:375]: Can we rename it to list-active-connectivity-config

File: src/network-manager/azext_network_manager/tests/latest/test_network_scenario.py [hunk 5]
```diff
@@ -374,6 +374,7 @@ def test_network_manager_list_queries(self, virtual_network, resource_group):
...
         self.cmd('network manager list-effective-config --virtual-network-name {virtual_network} -g {rg}')
```
Comment (yanfa317) [inline src/network-manager/azext_network_manager/tests/latest/test_network_scenario.py:376]: Can we rename it to list-effective-connectivity-config

**Comment by yanfa317** [comment]
I left some comments, except them, it looks good to me. Thanks!

File: src/network-manager/HISTORY.rst [hunk 6]
```diff
@@ -3,6 +3,15 @@
 Release History
 ===============
 
+0.1.1
```
Comment (kairu-ms) [inline src/network-manager/HISTORY.rst:6]:
```suggestion
0.2.0
```

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**